SeaSharpe_CVGS/SeaSharpe_CVGS/Migrations/Configuration.cs

```diff
@@ -195,98 +200,98 @@ protected override void Seed(ApplicationDbContext context)
-
-            /*
-            if (aprover == null)
-            {
-                order.Aprover = null;
-            }
-            else
-            {
-                order.Aprover = 
-            }*/
-
-            if (order.OrderItems == null)
-            {
-                order.OrderItems = new List<OrderItem>();
-            }
-
-            foreach (string game in games)
-            {
-                var currGame = db.Games.First(g => g.Name == game);
-                order.OrderItems.Add(new OrderItem { Order = order, Game = currGame, SalePrice = currGame.SuggestedRetailPrice });
-            }
-
-            return order;
+        }
+
+
+
+        /// <summary>
+        /// Use this method to create a mock addresses
+        /// </summary>
+        /// <param name="member">Username of the ApplicationUser</param>
+        /// <param name="addressName">address</param>
+        /// <param name="city">city name</param>
+        /// <param name="region">region(/province)</param>
+        /// <param name="country">country name</param>
+        /// <param name="postalCode">postal code</param>
+        /// <returns>Address Object</returns>
+        private Address MakeAddress(string member, string addressName, string city, string region, string country, string postalCode)
+        {
+            Address address = new Address
+            {
+                Member = db.Members.First(m => m.User.UserName == member),
+                StreetAddress = addressName,
+                City = city,
+                Region = region,
+                Country = country,
+                PostalCode = postalCode
+            };
+
+            return address;
+        }
+
+        /// <summary>
+        /// Use this method to create a mock order
+        /// </summary>
+        /// <param name="member">memberId</param>
+        /// <param name="aprover">approverId (null if unprocessed)</param>
+        /// <param name="orderPlacementDate">order placement date (null for shopping cart)</param>
+        /// <param name="shipDate">date that it was shipped out (null if not yet shipped)</param>
+        /// <param name="billingAddressIndex">index of billing address</param>
+        /// <param name="shippingAddressIndex">index of shipping address</param>
+        /// <param name="isProcessed">true if it has been processed</param>
+        /// <param name="games">name of all games in the order</param>
+        /// <returns>Order Object</returns>
+        private Order MakeOrder(string member, string aprover, string orderPlacementDate, string shipDate, int billingAddressIndex, int shippingAddressIndex, bool isProcessed, params string[] games)
+        {
+            Order order = new Order
+            {
+                Member = db.Members.First(m => m.User.UserName == member),
+                Aprover = db.Employees.FirstOrDefault(e => e.User.UserName == aprover),
+                BillingAddress = db.Addresses.First(b => b.Id == billingAddressIndex),

```

Comment:
The billing & shipping address indexes are being used like an ID. This is throwing an exception when the IDs are not found (and none of them would be since Address IDs are now 700000000+